core/src/main/java/com/amazon/opendistroforelasticsearch/sql/data/model/ExprStringValue.java

```diff
@@ -43,6 +48,40 @@ public String stringValue() {
     return value;
   }
 
+  @Override
```

Comment:
could you add the test cases for these functions?

Reply:
added test cases for date-time functions in ExprStringValue